doc/src/MainPage.dox

```diff
@@ -181,15 +183,17 @@
  *   "HostInterface" represents the caller of the API. It allows the
  *   asset management system to customize the @ref publish "publishing"
  *   process if desired and/or query additional information about the
- *   host and its documents.
+ *   host.
+ *
+ * The interface implementations are "wrapped" by the
```

Comment:
[consider][minor]
```suggestion
 * The interface implementations are "wrapped" in
```

Reply:
Hows [this](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/362/commits/d2c9966f1dd046926aa5c058e6810cb506d92e00)?